services/mailer.js

```diff
@@ -14,7 +8,7 @@ if (!process.env.TALK_SENDGRID_APIKEY) {
         '///////////////////////////////////////////////////////////////');
 }
 
-const transporter = nodemailer.createTransport(sgTransport(options));
+const transporter = nodemailer.createTransport(`smtps://apikey:${process.env.TALK_SENDGRID_APIKEY}@smtp.sendgrid.net`);
```

Comment:
Instead of building the transport string like this, we should use an env var called TALK_SMTP_CONNECTION_STRING and pass that directly into createTransport().  This will allow us to use sendgrid or any other provider.